bundles/org.openhab.binding.radiobrowser/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="radiobrowser"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="radio">
+		<label>Radio Thing</label>
+		<description>Provides a way to interact with the Radio Browser database of stations</description>
+
+		<channels>
+			<channel id="country" typeId="country"/>
+			<channel id="state" typeId="state"/>
+			<channel id="language" typeId="language"/>
+			<channel id="genre" typeId="genre"/>
+			<channel id="station" typeId="station"/>
+			<channel id="stream" typeId="stream"/>
+			<channel id="name" typeId="name"/>
+			<channel id="icon" typeId="icon"/>
+		</channels>
+
+		<config-description>
+			<parameter name="filters" type="text" required="true">
+				<label>Filters</label>
+				<description>Enter your own custom filter starting with ?, to ensure a stream that works with your device</description>
+				<default>?hidebroken=true&amp;limit=1700&amp;reverse=true&amp;order=votes</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="clicks" type="boolean" required="true">
+				<label>Clicks</label>
+				<description>Send a click for every station you select. This helps to mark stations as popular and makes the
+					database more useful to other people.</description>
+				<default>true</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="languageCount" type="integer" min="1" required="true">
+				<label>Language Count</label>
+				<description>The minimum number of stations a language needs to have world wide before the binding lists it as an
+					option.</description>
```

Comment:
It might be me, as English is not my native language, but i cant understand what this means.

Reply:
The languages are pulled from their server and contain many typos as it is added by public end users. By using this we throw away some real languages that are not common to get rid of the typos.

English
Englsih
Englihs
American English
British Engligh
etc....

The API tells us how many stations each one has, so we can reduce the list down to just the main common languages.

Reply:
```suggestion
				<description>The minimum number of stations a language needs to have, before the binding lists it as an
					option.</description>
```

Reply:
If this is hard to understand, should it be made ADVANCED? The default works fine, so it's just for those that want a fast way to get a better result without having to resort to METADATA creating state options from scratch.

Reply:
Not that hard to understand, it was just the world wide part that made it (tome) a bit vague. Now that you explained it, it made sense. As that part doesn;t add much, i sugggest to remove those words (see previous comment)

Regarding the question to mark it advanced or not, it is up to you. For both there are some pros and cons. I don't have a specific opinion on this.